src/accordion/Accordion.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @overview An Accordion module comprised of sections that toggle open and close
+ * @module Accordion.js
+*/
+
+export const Accordion = {
+
+  accordionSection : `.js-accordion-section`,
+  accordionSectionTrigger: `.js-accordion-section-toggle`,
+  accordionSectionContent: `.js-accordion-section-content`,
+  openClass: `is-open`,
+
+  /**
+   * Inits accordion elements
+   * @returns {void}
+  */
+  init() {
+    this.accordionSections  = document.querySelectorAll(this.accordionSection);
+
+    [...this.accordionSections].forEach((el) => {
+      let trigger = el.querySelectorAll(this.accordionSectionTrigger);
```

Comment:
@ckeisato You could just use `el.querySelector(this.accordionSectionTrigger);` instead of having to do line 22